exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -435,6 +435,10 @@
             let rowElem = $(shutdownBtn).parent().parent();
             let hostAddr = $(rowElem).find('#address').contents().get(0).nodeValue.trim();
             let hostPort = $(rowElem).find('#httpPort').html();
+            // Always use the host address from the url for the current Drillbit
```

Comment:
please add explanation or say `For details refer DRILL-6663`

Reply:
Done.